CefSharp/DependencyChecker.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright © 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// DependencyChecker provides a known list of Cef/CefSharp dependencies and 
+    /// provides helper methods to check for their existance.
+    /// </summary>
+    public static class DependencyChecker
+    {
+        public const string LocalesPackFile = @"locales\en-US.pak";
+
+        /// <summary>
+        /// List of Cef Dependencies
+        /// </summary>
+        public static string[] CefDependencies =
+        {
+            // CEF core library
+            "libcef.dll",
+            // Unicode support
+            "icudtl.dat"
+        };
+
+        /// <summary>
+        /// List of Cef Resources (pack files)
+        /// </summary>
+        public static string[] CefResources =
+        {
+            // Pack Files
+            // Note: Contains WebKit image and inspector resources.
+            "devtools_resources.pak",
+            "cef.pak",
+            "cef_100_percent.pak",
+            "cef_200_percent.pak"
+        };
+
+        public static string[] CefOptionalDependencies =
+        {
+            // Angle and Direct3D support
+            // Note: Without these components HTML5 accelerated content like 2D canvas, 3D CSS and WebGL will not function.
+            "libEGL.dll",
+            "libGLESv2.dll",
+            "d3dcompiler_43.dll",
```

Comment:
Even thought we don't care yet, we should probably comment about the XP vs Vista+ thing here.